docs/changelog.md

```diff
@@ -16,17 +16,43 @@ encapsulated this logic;
 - Constructors for ScMemoryContext with string and int parameters were removed. Use the one without parameters instead.
-- HelperSearchTemplateInStruct was removed from ScMemoryContext API. Use HelperSearchTemplate with callbacks.
+- HelperSearchTemplateInStruct was removed from ScMemoryContext API. Use SearchByTemplate with callbacks.
 - Deprecated sc-utils in 0.9.0 were removed from sc-kpm.
 - Questions were renamed to actions, answers were renamed to results.
 - `m_memoryCtx` in ScAgent renamed to `m_context`.
 - Now we don't use code generation as metaprogramming. The API for agents has been completely redesigned. See the documentation section on how to implement an agent with the new API. We have
   - completely moved from code generation to template programming;
   - improved the API and aligned with our description of how it should be;
   - simplified the API. Now it will be much easier to create agents, go right now and see how to work with the new API -- [C++ Agents API](sc-memory/api/cpp/extended/agents/agents.md).
+- We redesigned all `ScMemoryContext` methods to a common style. All methods with not correct names were deprecated, and new ones were added. See the table below to see which methods have been replaced.
+  
+  | Deprecated method                   | Substitution method                   |
+  |-------------------------------------|---------------------------------------|
+  | CreateNode                          | GenerateNode                          |
+  | CreateLink                          | GenerateLink                          |
+  | CreateEdge                          | GenerateConnector                     |
+  | GetEdgeSource                       | GetArcSourceElement                   |
+  | GetEdgeTarget                       | GetArcTargetElement                   |
+  | GetEdgeInfo                         | GetConnectorIncidentElements          |
+  | HelperCheckEdge                     | CheckConnector                        |
+  | FindLinksByContent                  | SearchLinksByContent                  |
+  | FindLinksByContentSubstring         | SearchLinksByContentSubstring         |
+  | FindLinksContentsByContentSubstring | SearchLinksContentsByContentSubstring |
+  | HelperSetSystemIdtf                 | SetElementSystemIdentifier            |
+  | HelperGetSystemIdtf                 | GetElementSystemIdentifier            |
+  | HelperResolveSystemIdtf             | ResolveElementSystemIdentifier        |
+  | HelperFindBySystemIdtf              | SearchElementBySystemIdentifier       |
+  | HelperGenTemplate                   | GenerateByTemplate                    |
+  | HelperSearchTemplate                | SearchByTemplate                      |
+  | HelperBuildTemplate                 | BuildByTemplate                       |
+  | CalculateStat                       | CalculateStatistics                   |
+  | BeingEventsPending                  | BeginEventsPending                    |
```

Comment:
What about event blocking?

Reply:
It was added in the current version

Reply:
did you forget to commit line with event blocking?